### PR TITLE
client fixes

### PR DESCRIPTION
**Pull request by diego-vieira**

### CODE REVIEW

File: Procfile [hunk 1]
```diff
@@ -1 +1 @@
-web: node src/
\ No newline at end of file
+web: NODE_ENV=production node src/
```
Comment (diego-vieira) [inline Procfile]: Required when deploying to Heroku

File: src/index.js [hunk 2]
```diff
@@ -1,9 +1,9 @@
 'use strict';
 
 const app = require('./app');
-const port = app.get('port');
+const port = process.env.PORT || app.get('port');
```
Comment (diego-vieira) [inline src/index.js]: Required when deploying to Heroku

File: Procfile (hunk 1 above)
Comment (daffl) [inline Procfile]: Ah, I usually set this in the environment configuration but this makes much more sense.

**Comment by daffl** [comment]
This are some great fixes. Thank you!

File: src/index.js (hunk 2 above)
Comment (ekryski) [inline src/index.js]: Does this not already get pulled in by feathers-configuration if you have `PORT` set as an environment var?

**Pull request merged**